README.md

```diff
@@ -249,19 +256,20 @@ export SERVICE_TRIGGER_ADDR=`jq -r .deployedTo .docker/trigger.json`
 Deploy the compiled component with the contract information from the previous steps. Review the [makefile](./Makefile) for more details and configuration options.`TRIGGER_EVENT` is the event that the trigger contract emits and WAVS watches for. By altering `SERVICE_TRIGGER_ADDR` you can watch events for contracts others have deployed.
 
 ```bash docci-delay-per-cmd=3
-export COMPONENT_FILENAME=evm_price_oracle.wasm
 
-# === LOCAL ===
-export IS_TESTNET=false
-export WASM_DIGEST=$(make upload-component COMPONENT_FILENAME=$COMPONENT_FILENAME)
-
-# === TESTNET ===
-# ** Setup: https://wa.dev/account/credentials
-export IS_TESTNET=true
-export PKG_VERSION="0.1.0"
-export PKG_NAMESPACE=`warg info --namespaces | grep = | cut -d'=' -f1 | tr -d ' '`
-export PKG_NAME="${PKG_NAMESPACE}:evmpriceoraclerust"
-warg publish release --name ${PKG_NAME} --version ${PKG_VERSION} ./compiled/${COMPONENT_FILENAME} || true
+
+export COMPONENT_FILENAME=evm_price_oracle.wasm
+if [ "$DEPLOY_ENV" = "LOCAL" ]; then
+    # TODO: would require WAVS to start here until we can get
+    # it to move to the else block here. then we could keep wavs off for now
+    export WASM_DIGEST=$(make upload-component COMPONENT_FILENAME=$COMPONENT_FILENAME)
+else
+    # ** Setup: https://wa.dev/account/credentials
+    export PKG_VERSION="0.1.0"
+    export PKG_NAMESPACE=`warg info --namespaces | grep = | cut -d'=' -f1 | tr -d ' '`
+    export PKG_NAME="${PKG_NAMESPACE}:evmpriceoraclerust"
+    warg publish release --name ${PKG_NAME} --version ${PKG_VERSION} ./compiled/${COMPONENT_FILENAME} || true
+fi
```

Comment:
This is so ugly

![image](https://github.com/user-attachments/assets/81764a78-a529-4588-9481-6d10bf9e4272)